generator/mavgen_cs.py

```diff
@@ -328,7 +328,8 @@ def generate(basename, xml):
     
     generatedCsFiles = [ serfilename, structsfilename]
     
-    includedCsFiles =  [ 'CS/common/ByteArrayUtil.cs', 'CS/common/FrameworkBitConverter.cs', 'CS/common/Mavlink.cs'  ]
+    cur_dir = os.path.dirname(os.path.abspath(__file__)) + '/'
```

Comment:
This should be `os.path.join`

This should also look something like this:
```
     includedCsFilesCommon =  [ 'ByteArrayUtil.cs', 'FrameworkBitConverter.cs', 'Mavlink.cs'  ]
     cur_dir = os.path.dirname(os.path.abspath(__file__))
     incudeCSFiles = [ os.path.join([cur_dir, "CS", "Common", x]) for x in includedCsFilesCommon ]
```

... but I'm only really fussed with using the appropriate join function.